autogtp/README.md

```diff
@@ -16,6 +16,13 @@ the SGF and training data at the end of the game.
     sudo apt install qt5-default qt5-qmake curl
     qmake -qt5
     make
+    
+## Example of compiling - macOS
+
+	brew install qt5
+	brew link qt5 --force
+	qmake
```

Comment:
The brew output shows
```
==> Caveats
This formula is keg-only, which means it was not symlinked into /usr/local,
because Qt 5 has CMake issues when linked.
```
Not clear if we should be having people force link it. Maybe just `/usr/local/opt/qt/bin//qmake` instead?

Reply:
I believe the better fix is:

```
export PATH=/usr/local/opt/qt5/bin:$PATH
```

Before doing the cmake .. to generate the build files. (If you already did, you have to remove everything or the above won't work).

Reply:
Oh, this is for compiling autogtp directly. Yeah, then what @Mardak says is preferable.